modules/AlphaWalletFoundation/AlphaWalletFoundation/Tokens/Logic/GetContractInteractions.swift

```diff
@@ -21,11 +21,23 @@ class GetContractInteractions {
         let request = GetContractList(walletAddress: walletAddress, server: server, startBlock: startBlock, erc20: erc20)
         return networkService
             .dataTaskPublisher(request)
+            .handleEvents(receiveOutput: { self.log(response: $0) })
```

Comment:
Can shorten it a little to:

```
.handleEvents(receiveOutput: self.log)
```

Reply:
@oa-s not super important, but seems not included. In case you pushed the wrong commit…